### PR TITLE
feat(profile): add Messaging settings view in profile

### LINKED ISSUE #4931: New Messaging Settings
### Description

design: https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=1511%3A137582

Screen Shot 2022-02-28 at 5 04 47 PM

Most sections already exist and can be moved but the design needs to be adjusted.

Todo:
- [x] Add "Messaging" section to Apps section
- [x] Badge should reflect total number of pending contact requests
- [x] Move "allow new contact request" from Privacy and security section
- [x] Add "Show My Profile Picture To". note: "Only Verified Contacts is out of scope". This api already exists in status-go.
- [x] Move "Open message links with" from Privacy and security section
- [x] Move "Display Message Link Previews" from Privacy and security section
- [x] Move Mailserver from Sync settings section
- [x] Remove Sync settings setction
- [x] Move Contacts from Contacts section note: identity verified contacts are out of scope

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #4931

image
image
image
image
image
image
image

Note that the Contact view is not final, it's quite an undertaking so it should be done in another PR, because this one is quite massive already

StatusQ PR https://github.com/status-im/StatusQ/pull/573

### CODE REVIEW

File: src/app_service/service/settings/dto/settings.nim [hunk 1]
```diff
@@ -166,6 +166,7 @@ proc toWalletVisibleTokens*(jsonObj: JsonNode): Table[int, seq[string]] =
       result[chainId].add(token.getStr)
 
 proc toSettingsDto*(jsonObj: JsonNode): SettingsDto =
+  echo jsonObj
```
Comment (alaibe) [inline src/app_service/service/settings/dto/settings.nim:169]: oops ;)

**Review by alaibe — APPROVED** [review]

File: src/app_service/service/settings/dto/settings.nim (hunk 1 above)
Comment (jrainville) [reply src/app_service/service/settings/dto/settings.nim:169]: 🤫

**Review by iurimatias — CHANGES_REQUESTED** [review]
The Messaging section should be in the Apps group

**Review by anastasiyaig — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]
tested

**Pull request merged**